### PR TITLE
add answer_choices_key for quartz

### PR DESCRIPTION
**Pull request by VictorSanh**

### CODE REVIEW

File: promptsource/templates/quartz/templates.yaml [hunk 1]
```diff
@@ -145,17 +133,15 @@ templates:
       - Sequence Accuracy
       original_task: true
     name: having_read_above_passage
-    reference: 'Comment: Answer_choices_key: choices["text"]'
+    reference: ''
   a8c6ae4c-4874-47d1-93ea-801b6e080a58: !Template
     answer_choices: null
-    answer_choices_key: null
+    answer_choices_key: '{{choices.text | join("|||")}}'''
```
Comment (stephenbach) [inline promptsource/templates/quartz/templates.yaml:139]: Extra single quote

File: promptsource/templates/quartz/templates.yaml [hunk 2]
```diff
@@ -40,18 +36,16 @@ templates:
       - Sequence Accuracy
       original_task: true
     name: paragraph_question_plain_concat
-    reference: 'Comment: Answer_choices_key: choices["text"]'
+    reference: ''
   22e29cab-f57f-4af7-92fc-72b131a96878: !Template
     answer_choices: null
-    answer_choices_key: null
+    answer_choices_key: '{{choices.text | join("|||")}}'''
```
Comment (stephenbach) [inline promptsource/templates/quartz/templates.yaml:42]: Extra single quote.

**Review by stephenbach — APPROVED** [review]
LGTM! Just an extra single quote in a couple of the answer choices key that is throwing off the text

**Comment by VictorSanh** [comment]
ooops, indeed. fixed it, merging now

**Pull request merged**